cpp/log/test.cpp

```diff
@@ -61,4 +68,76 @@ void test() {
 	t1.join();
 }
 
+/**
+ * Creates a FileSink to also write log Messages
+ * into /tmp/openage-log-test.
+ * Only Messages with a level equals or higher
+ * than level::err will be printed in that file.
+ */
+void file_log_sink () {
+	const char *filelog_name = "/tmp/openage-log-test";
```

Comment:
Hey @mic-e ! I am trying to use mkstemp to create a temporary file (it has a random name /tmp/file.XXXXXX) but I can't find a way to retrieve that random name so I can give it to the FileSink. What were you exactly thinking about mkstemp ? or maybe I am missing something.